docs/config.c

```diff
@@ -2210,6 +2210,43 @@
 ** how often (in seconds) NeoMutt will update message counts.
 */
 
+{ "mailbox_folder_format", DT_STRING, "%2C %t %N %n %m %i" },
```

Comment:
If we go for a default, I'd avoid duplicating the new info with both %N and %n, how about a conditional, e.g. `%2C %?n?%6n&%6 ? %6m %i`?

Reply:
yes, that looks better

Reply:
I'm good with `%2C %?n?%6n&%6 ? %6m %i`. I did suggest with `%N` because it's easier *for me* to spot folders with new mail.